test/e2e/tests/risingwave/meta.sh

```diff
@@ -14,28 +14,55 @@
 
 ${__E2E_SOURCE_TESTS_RISINGWAVE_META_SH__:=false} && return 0 || __E2E_SOURCE_TESTS_RISINGWAVE_META_SH__=true
 
-function risingwave::utils::kill_the_meta_leader_pod() {
-  local meta_names
-  meta_names="$(k8s::kubectl::get pod -l risingwave/component=meta -o=jsonpath='{.items..metadata.name}')"
-  if [ -z "$meta_names" ]; then
-    logging::error "failed to retrieve meta nodes"
+function risingwave::utils::delete_leader_lease() {
+  local meta_leaders_pod_IPs
+  meta_leaders_pod_IPs="$(k8s::kubectl::get pod -l risingwave/component=meta -l risingwave/meta-role=leader -o=jsonpath='{.items..status.podIP}')"
+
+  if [ -z "$meta_leaders_pod_IPs" ]; then
+    logging::error "No meta leader node found"
+    return 1
+  fi
+
+  if [ "$(echo "$meta_leaders_pod_IPs" | wc -l)" -gt 1 ]; then
+    logging::error "More than one meta leader node found"
     return 1
   fi
 
-  local leader=""
-  for p in $meta_names; do
-    local is_leader
-    is_leader=$(k8s::kubectl logs "$p" | grep "Defining leader services")
-    if [ "$is_leader" != "" ]; then
-      if [ "$leader" != "" ]; then
-        logging::error "Split brain detected! $p and $leader are leaders!"
-        return 1
-      fi
-      leader="$p"
+  # Iterate over the etcd election kv pairs. Delete leader lease if found, else abort the test
+  local del_lease=false
+  while read -r i ; do
+    if [ "$(echo "$i" | jq -r .value | base64 --decode)" = "${meta_leaders_pod_IPs}:5690" ] ; then
+      del_lease=true
+      echo "found leader lease. Deleting it"
+      # delete leader lease
+      k8s::kubectl exec -it etcd-0 -- env ETCDCTL_API=3 etcdctl del "$(echo "$i" | jq -r .key | base64 --decode)" --user=root:iGGUrEKBiX
```

Comment:
The `iGGUrEKBiX` is the password from my local... I presume it won't work in the tests. Could you try removing the `--user` and see if it works?